app/uk/gov/hmrc/nisp/controllers/TermsConditionsController.scala

```diff
@@ -29,9 +29,9 @@ object TermsConditionsController extends TermsConditionsController with Authenti
 
 trait TermsConditionsController extends NispFrontendController with Actions {
 
-  def show(showBackLink: Option[Boolean] = Some(false)): Action[AnyContent] = UnauthorisedAction(
+  def show: Action[AnyContent] = UnauthorisedAction{
     implicit request =>
-      Ok(termsAndConditions(showBackLink.getOrElse(false)))
-  )
-
+      val showBackLink = request.queryString.get("showBackLink").getOrElse(List("false")).head.toBoolean
```

Comment:
`request.queryString.get("showBackLink").fold(false)(_.head.toBoolean)` would be more appropriate as it does not involve List("false") which is a bit misleading